TouchPortal-API/TouchPortalAPI/__init__.py

```diff
@@ -102,13 +102,11 @@ def __run(self):
 
     def __onReceiveCallback(self, rawData: bytes):
         data = json.loads(rawData.decode())
-        try:
-            if data['type'] == "down":
+        if (act_type := data.get('type')):
```

Comment:
The "walrus" is Python 3.8+. Not sure if that's an issue for anyone...  It could easily be two lines instead.